### ISSUE TITLE
Failing initialization when appliance is offline

### ISSUE THREAD

**Issue by ottes** [ofalvai/home-assistant-candy#288]
Hey,
may or may you not integrate some backoff mechanism, that does not make the complete intrgration fail, but only the offline device? 
Like sync initialization an async the devices?
I'm not that python experienced, but i may try myself, if you are out of time (when i have enough time these days) and provide your a PR maybe?

best regards,
Michael

**Comment by d13g0m0nt3s** [comment]
Hi. I agree with you. Hope the developer can implement  if he have the time. And offline will be better then the error message.

**Comment by Erreur32** [comment]
Yes good point, if possible to implement that would be better ;)

**Comment by basilottac** [comment]
Same for me please.

**Comment by ottes** [comment]
Any news here? Maybe someone knows a fork? I'm not into python that much, can't be such a thing, i guess
